practice-problems/lesson1.js

```diff
@@ -126,25 +131,25 @@
 
 //    console.log(x !== y);
 
-//Answer:
+//Answer:true
 
 
 
 
 //20. What’s the difference between == and ===?
 
-//Answer:
+//Answer: == not equal to,  ===equal to
```

Comment:
== checks for equivalency in value *only*.

=== checks for equal value *and* data type.